### PR TITLE
master-lrac-8979

### PR DESCRIPTION
**Pull request by suilin**
https://issues.liferay.com/browse/LRAC-8979
https://issues.liferay.com/browse/LRAC-8980
https://issues.liferay.com/browse/LRAC-8981
https://issues.liferay.com/browse/LRAC-8982
https://issues.liferay.com/browse/LRAC-8983

### CODE REVIEW

**Comment by suilin** [comment]
ci:test:sf

**Comment by christianstokes** [comment]
cc @alexcandidos @ViniLopes20 please note, the expected behavior for the View All Interests test is based on the conversation here in this thread: https://liferay.slack.com/archives/C028J6BS3SP/p1630341805057300?thread_ts=1630313705.050900&cid=C028J6BS3SP

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsInterests.testcase [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		ACUtils.tearDownDXP();
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+
+		ACUtils.tearDownAC();
+	}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsInterests.testcase:39]: Hi @suilin, Could you put your tearDown to delete the created sites? 

```
tearDown {
	var testPortalInstance = PropsUtil.get("test.portal.instance");

	ACUtils.tearDownDXP();

	if ("${testPortalInstance}" == "true") {
		PortalInstances.tearDownCP();
	}
	else {
		JSONGroup.deleteGroupByName(groupName = "Site Name");
	}

	ACUtils.tearDownAC();
}
```

Comment (suilin) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsInterests.testcase:39]: Hi @ViniLopes20, sure, I will update it.

**Comment by suilin** [comment]
ci:test:sf

**Comment by suilin** [comment]
ci:test:sf

**Comment by suilin** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACInterests.macro [hunk 2]
```diff
@@ -36,12 +36,36 @@ definition {
...
+	macro sortIndividualInterests {
+		Click(locator1 = "ACInterests#INTEREST_FILTER");
+
+		if (isSet(desc)) {
+			AssertElementPresent(locator1 = "ACInterests#INTEREST_FILTER_ARROW_DESC");
+		}
+		else {
+			AssertElementPresent(locator1 = "ACInterests#INTEREST_FILTER_ARROW_ASC");
+		}
+	}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACInterests.macro:48]: Hi @suilin, I think this `ACUtils.orderArrow` macro does the same function that your macro. What do you think?

Comment (suilin) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACInterests.macro:48]: Hi @ViniLopes20, they are not exactly the same, the `ACUtils.orderArrow` only work on 
`Individuals > Known Individuals > ac ac  > Interests`, ACInterests.sortIndividualInterests can work both on `Individuals > Known Individuals > ac ac  > Interests` and `Individuals > Interests`. Thank you!

**Comment by suilin** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @suilin , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**